src/test/a11y/test-suite.js

```diff
@@ -10,14 +10,7 @@ const { URLS, BASE_URL } = require('./paths')
 
 const FIRST_APPLY_PAGE_URL = URLS['DO_YOU_LIVE_IN_SCOTLAND']
 
-const runAndAggregateIssueChecks = (url, result) => (results, checkFn) => {
-  const resultOfCheck = checkFn(url, result)
-  if (resultOfCheck !== null) {
-    return [...results, resultOfCheck]
-  }
-
-  return results
-}
+const notNull = result => result != null
```

Comment:
We have a `notIsNil` method in `predicates.js` that we could use instead of this.